opengrok-indexer/src/main/java/org/opengrok/indexer/index/Filter.java

```diff
@@ -216,6 +216,8 @@ private Pattern compilePattern(String pattern) {
     }
 
     public static class PatternList extends ArrayList<String> {
+        private static final long serialVersionUID = -6883390970972775838L;
```

Comment:
Missing space between class and attribute.

Reply:
is there a way how to enforce this in style checker ?

Reply:
> is there a way how to enforce this in style checker ?

I'm not sure. Maybe somehow using regular expressions?